packages/blob/src/multipart/upload.ts

```diff
@@ -52,13 +50,13 @@ export function createUploadPartMethod<
   };
 }
 
-export function uploadPart({
+export async function uploadPart({
   uploadId,
   key,
   pathname,
   headers,
   options,
-  abortController,
+  abortController = new AbortController(),
```

Comment:
Should we name this internalAbortController so we can easily distinguish it from a user abortController when reading the code?